### PR TITLE
Readme branch to be merged in development

### PR DESCRIPTION
**Pull request by Annel18**
First PR request for testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,230 @@
...
+<!-- USAGE EXAMPLES -->
+## Usage
+
+Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+
+_For more examples, please refer to the [Documentation](https://example.com)_
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```
Comment (rasimsen) [inline README.md:125]: can you add some testing info? which library, how to run tests, what was the test coverage? do you need a mocking? which mocking library , how will you test third-party rest-apis (just testing you won't hit the real rest endpoint I guess) etc..

File: README.md [hunk 2]
```diff
@@ -1,8 +1,230 @@
...
+
```
Comment (rasimsen) [inline README.md:126]: CI/CD info , please?

Comment (rasimsen) [inline README.md:126]: Prod server info please?

Comment (Annel18) [reply README.md:164]: CI/CD workflow stastus badges added

File: README.md (hunk 1 above)
Comment (Annel18) [reply README.md:160]: New testing section added

File: README.md [hunk 3]
```diff
@@ -1,8 +1,274 @@
...
+    API_KEY = 'ENTER YOUR API';
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
```
Comment (rasimsen) [inline README.md:121]: can you add how to run in local, dev, prod, please?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,274 @@
...
+Use this space to show useful examples of how a project can be used.
```
Comment (rasimsen) [inline README.md:122]: can you add how to test over jest with test coverage, please?

File: README.md (hunk 3 above)
Comment (Annel18) [reply README.md:130]: added info how to run in local and prod

**Pull request merged**